### ISSUE TITLE
Promoting AntiMicroX

### ISSUE THREAD

**Issue by pktiuk** [juliagoda/antimicroX#130]
Hey,

I think AntiMicroX is a very good project which doesn't get deserved attention.
A lot of work has been done to original AntiMirco, and it could currently without any problems replace original Antimicro which is still quite popular.
- [downloads from github](https://somsubhra.com/github-release-stats/?username=antimicro&repository=antimicro) approx 350k overall
- [downloads from SourceForge](https://sourceforge.net/projects/antimicro.mirror/files/stats/timeline) approx 18k this week
Compared with AntiMicroX [clones](https://somsubhra.com/github-release-stats/?username=juliagoda&repository=antimicrox) and [star history](https://star-history.t9t.io/?ref=producthunt#juliagoda/antimicroX)

But there are some problems with this app which are contributing to low popularity and bad accessibility.

In my opinion, they are:
1. It is difficult to find it - it would be good if AntiMicroX would be easier to find by someone who is looking for this type of software. It should be searchable at least on sites like [AlternativeTo](https://alternativeto.net/software/antimicrox/) or [SourceForge](sourceforge.net). Maybe we should also promote it a bit there (add something like "If you liked an app then like it there" and put it in README, or maybe in "About" window)
2. It is a very inaccessible - there is no ready to install package for Windows or macOS (it would be good to have also `rpm` package), most of the users have to compile it manually (it can be very difficult for people who are not so tech-savvy)
3. Its site is not very visually compelling - I think we should change README a bit (top part could contain some screens and installation guide) and we should change the icon, because it has low resolution and isn't very clean (as most of minimalistic and modern icons today). We could use something from [flaticon](www.flaticon.com), icons from there have a license that allows us to use them in our project (even if this project would be commercial).

Things (I think) should be done for now:
- [x] select new icon for an app
- [x] finish AntiMicroX's site on [AlternativeTo](https://alternativeto.net/software/antimicrox/) and (maybe) edit AntiMicro's page
- [ ] add AntiMicroX to [SourceForge](sourceforge.net) and (maybe) edit AntiMicro's page
- [ ] add installer for Windows: https://github.com/juliagoda/antimicroX/issues/131
- [ ] (optional) add portable version for Windows (it was implementad in old AntiMicro [link](https://github.com/AntiMicro/antimicro/blob/master/BuildOptions.md))
- [ ] add installer for macOS
- [ ] change README (add simple installation instructions, and maybe some screenshots of an app)

I think some of these tasks should get their own issues.

What do you think about these tasks? Do you have any comments? Ideas? Something else should be done?

**To sum up:**
If you want to help you can:
- Like AntiMicroX's on [AlternativeTo](https://alternativeto.net/software/antimicrox/)
- Consider helping us with one of tasks mentioned here (currently Windows port is our top priority)

**Comment by pktiuk** [comment]
I think, that our first task should be changing icon.

These my favourite candidates:
<table border="0px" ><tr><td width = 400px>
https://www.flaticon.com/free-icon/gamepad_2991606

<img src="https://image.flaticon.com/icons/svg/2991/2991606.svg" width=200px% >
</td><td>
https://www.flaticon.com/free-icon/gamepad_957230?term=gamepad&page=2&position=14
<img src="https://image.flaticon.com/icons/svg/957/957230.svg" width=200px% >
</td></tr></table>

They are customizable, so we could tweak colors a bit.
(I think the left one looks better)

**Comment by AliAsiri20** [comment]
Hi, 
I'm Ali Asiri from Saudi Arabia. I've been watching AntimicroX closely. I've downloaded AX 3.0 in the first 18 hours of release.
Antimicro is the best game pad mapper for me, and I think AntimicroX is better. I've tried all popular gamepad mappers including commercial ones. Non of them is doing the mapping job like Antimicro. I'm a Windows user, and I've never used Linux in my life, although I like everything about it. But, I'm not a programmer.

I thought about following CMake instructions and install or compile AntimicroX in my Win PC. But that wasn't a good Idea, man, I downloaded almost 10GigB of stuff that are "needed" to compile a c++ application. And none of the instructions I followed worked.

I signed up to GitHub just to reply to your thread, and I really wanted to say thank you for keeping this great app alive. For now, I'm using other mappers because Antimicro that works in windows has too much issues.

**Comment by malvarenga123** [comment]
I don't know if many people do this, but if a project hasn't been updated in a long time, I take a look at the fork graph that github provides. Usually you can find the project that has taken over by the number and by how recent the commits are.
This project, however, was not forked, so I had no idea it existed. I only found out because you put a warning in the Issues tab of the original project.
Maybe by forking it from the original one you'll get other people to see that development has continued.

**Comment by pktiuk** [comment]
@malvarenga123 
I have found this repo by an accident, and it had good content but was practically inaccessible for regular users. That's why I do everything to make it easier to get. (I have added configuration for building packages for Linux and automated releasing) and now I opened this issue because of this.

@AliAsiri20 
I have already opened an issue for this: https://github.com/juliagoda/antimicroX/issues/131
It will be more difficult than doing it on Linux because Windows is much less programmer-friendly.

For now, you could also fill the survey, I have put in the comment with icons.
And like AntiMicroX on [AlternativeTo](https://alternativeto.net/software/antimicrox/)

**Comment by AliAsiri20** [comment]
Thank you pktiuk.
I did the survey. I liked Icon3 because it's almost square, fits good with other icons, and the bright color would stand out very well in any theme color. Icon1 is also very good but its dark color will make it hard to see in dark themes.

I also liked AntimicroX on AlternativeTo.

**Comment by pktiuk** [comment]
First icon won. 
I have opened PR changing this icon (and adding scalable version of it). : https://github.com/juliagoda/antimicroX/pull/132
![image](https://user-images.githubusercontent.com/45544416/85956640-6ab21280-b987-11ea-9f98-5915915a5959.png)

**Comment by juliagoda** [comment]
> Maybe by forking it from the original one you'll get other people to see that development has continued.

Because I couldn't get along with the previous developer, with the help of the mirroring command I moved the whole story, so that in the "Insights" tab you can see the whole coherence. In my opinion, neat overall changes are more important than just forking. There's no point in creating a fork, if this program is more and more different from the original and even more so, if the program has a whole lot of forks on its website (there are dozens of them), how will you know which one is the most suitable? Are you browsing through dozens of forks one by one? How many people could want to do that?

**Comment by pktiuk** [comment]
This is the whole point of creating fork is having some integrity through showing source of this software. When I find an abandoned project I always check forks because this is the easiest way of finding somebody who is still developing this project.

> how will you know which one is the most suitable? Are you browsing through dozens of forks one by one? How many people could want to do that?

I use [this tool](https://techgaun.github.io/active-forks/index.html) to check which fork has the latest changes.

**Comment by sirlucjan** [comment]
> This project, however, was not forked, so I had no idea it existed. I only found out because you put a warning in the Issues tab of the original project.
> Maybe by forking it from the original one you'll get other people to see that development has continued.

I can guarantee that more than 90% of potential users of this program have no idea how github works and that there is such a thing as fork. I'll even say more - they have no idea how to build such a program and if someone doesn't deliver it to them in some repository, they'll get stuck.

**Comment by juliagoda** [comment]
By the way please remove the tag "Windows" on AlternativeTo and leave only Linux, if it's possible. It's not a true so far

**Comment by michaelsjackson** [comment]
> Hi,
> I'm Ali Asiri from Saudi Arabia. I've been watching AntimicroX closely. I've downloaded AX 3.0 in the first 18 hours of release.
> Antimicro is the best game pad mapper for me, and I think AntimicroX is better. I've tried all popular gamepad mappers including commercial ones. Non of them is doing the mapping job like Antimicro. I'm a Windows user, and I've never used Linux in my life, although I like everything about it. But, I'm not a programmer.
> 

You can use linux also without being a programmer. You should delete first Windows. Or at least start using linux in VirtualBox there. Have a nice day.

**Comment by pktiuk** [comment]
Due to ending maintenance of AntiMicroX on this repository this issue is moved to a new repository and will and can be further tracked here: https://github.com/AntiMicroX/antimicroX/issues/5
